### DISCUSSION TITLE
'Group' object has no attribute 'user_set'

### DISCUSSION THREAD

**Discussion by CTV-2023** [netbox-community/netbox#16098] [Help Wanted!] [answered]
### Deployment Type

Self-hosted

### NetBox Version

v4.0.0

### Python Version

3.11

### Steps to Reproduce

Hi,

Following this discussion
https://github.com/netbox-community/netbox/discussions/16000
I think I found a regression or missing feature between Django user/group models and Netbox models

If in a custom pipeline, calling netbox authentication model with 
`from netbox.authentication import Group`

I can't add users to a group with group.user_set.add(user)
> <class 'AttributeError'>

> 'Group' object has no attribute 'user_set'

I can still add groups to a user with user.groups.add(group)

(many to many relationship issue ?)

### Expected Behavior

I should be able to add users to a group, like I can add groups to user

### Observed Behavior

Regression from Django

**Comment by abhi1693** [comment, marked as answer]
You should use `group.users.add(user)`

**Reply by CTV-2023** [reply]
Thanks :)
